docs/docs/reference/config-files/gatsby-config.md

```diff
@@ -224,6 +226,48 @@ You can overwrite the search paths, which contain documents that should be scann
 
 By default, `graphqlTypegen` is only run during `gatsby develop`. Set this option to `true` to create the `src/gatsby-types.d.ts` file also during `gatsby build`. Default: `false`.
 
+## adapter
+
+> Support added in `gatsby@5.X.0`
+
+You can set an [adapter](/docs/how-to/previews-deploys-hosting/adapters/) or configure the [zero-configuration deployments](/docs/how-to/previews-deploys-hosting/zero-configuration-deployments/) through the `adapter` setting.
```

Comment:
I think we can remove 'the' since it is followed by the plural version of zero-config deployments?

```suggestion
You can set an [adapter](/docs/how-to/previews-deploys-hosting/adapters/) or configure [zero-configuration deployments](/docs/how-to/previews-deploys-hosting/zero-configuration-deployments/) through the `adapter` setting.
```